interfaces/automated-testing/scd/scd.yaml

```diff
@@ -36,6 +36,506 @@ components:
+            notifications for operational intents.
+          anyOf:
+          - $ref: '#/components/schemas/EntityID'
+        new_subscription:
+          description: >-
+            If an existing subscription is not specified in `subscription_id`,
+            and the operational intent is in the Activated, Nonconforming, or
+            Contingent state, then this field must be populated.  When this
+            field is populated, an implicit subscription will be created and
+            associated with this operational intent, and will generally be
+            deleted automatically upon the deletion of this operational intent.
+          anyOf:
+          - $ref: '#/components/schemas/ImplicitSubscriptionParameters'
+
+    FlightAuthorisationData:
+      description: |-
+        A dataset to hold details of a UAS flight authorization request matching some of the requirements of Annex IV of the U-Space regulation 2021/664
+      type: object
+      properties:
+        uas_serial_number:
+          type: string
+          example: "T4S6BBMSZV4H0650"
+          description: Serial number of the UAS per the CTA-2063-A standard
+        operation_mode:   
+          $ref: '#/components/schemas/OperationMode'
+        operation_category:
+          type: string
+          example: "normal"
+          description: |-
+            Specify if the flight is "special" or has a specific priority.
+        uas_class: 
+          $ref: '#/components/schemas/UASClass'
+        identification_technologies: 
+          type: array
+          items:
+            type: string
+            enum:
+              - network
+              - broadcast
+            description: Specify an identification technology that will be used for this flight
+        connectivity_methods:
+          type: array
+          items:
+            type: string
+          example: ['http://ussp.com/emergency']
+        endurance_minutes: 
+          type: number
+          description: A number to specify the endurance of the drone in minutes
+        emergency_procedure_url:
+          type: string
+          format: url
+          description: A valid url to contact in case of emergencies
+        operator_id:
+          type: string
+          example: "CHE0n4fujx7nwanb-fwi"
+          description: ID assigned to the operator per the EN4709-02 standard
+      required:
+      - uas_serial_number
+      - operation_mode
+      - uas_class
+      - identification_technologies
+      - operator_id
+
+    OperationMode:
+      type: string
+      enum:
+        - vlos
+        - bvlos
+      description: Specify if the operation is a `vlos` or `bvlos` operation
+
+    UASClass:
+      type: string
+      enum:
+        - C0
+        - C1
+        - C2
+        - C3
+        - C4
+      description: Specify the class of the UAS to be flown, the specifition matches EASA class identification label categories, for more information see EASA website
+
+    IdentificationTechnology:
```

Comment:
This should either be used with a `$ref` in the `items` of `identification_technologies` or removed

Reply:
I have now removed this.